sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_polling.py

```diff
@@ -7,7 +7,8 @@
 
 import datetime
 import json
-from typing import Union, TypeVar, Any
+from typing import Callable, Mapping, Union, TypeVar, Any, Optional
+from typing_extensions import Protocol
```

Comment:
I think this is the first time we've imported from typing_extensions in this library so we should add it to the setup.py dependencies